lib/core/src/animation/animator/continuous.rs

```diff
@@ -14,7 +16,7 @@ struct ContinuousTimeAnimatorProperties {
     callback          : Box<dyn AnimationCallback>,
     relative_start_ms : f32,
     absolute_start_ms : Option<f32>,
-    callback_guard    : Option<AnimationFrameCallbackGuard>
+    callback_guard    : Option<CallbackHandle>
```

Comment:
Maybe "event_loop_callback_guard" to distinct this from the animation callback

Reply:
@farmaazon thanks for pointing it out! The name certainly needs to be fixed.